### DISCUSSION TITLE
Assign modifier keys to mouse buttons

### DISCUSSION THREAD

**Discussion by 8Slain8** [linearmouse/linearmouse#828] [Ideas]
I'm a cutter and often I use all the possible mouse click/movement/wheel combinations with different modifier keys, sometimes even more than one keys combined, and i've tried setting up shift to one of my mouse buttons. 
unfortunately it didn't work in premiere with both option (repeating and not) 
is it an expected behaviour? would be great if something could be done about that
thx in advance and thanks for the great app =)

**Comment by ghost** [comment]
Similar issue here. I'm on macOS and I wanted to assign Scroll Down+CMD+CTRL to a hotkey in a specific application. 
I was unable to make this work in any scenario though :(

Also, ditto, thanks for the awesome app - couldn't live without it now!
